conanfile.py

```diff
@@ -48,25 +48,29 @@ def config_options(self):
             del self.options.fPIC
 
     def build_requirements(self):
-        if not self.options.with_system_deps: self.build_requires('protobuf/3.21.4')
-        if not self.options.with_system_deps: self.build_requires('grpc/1.48.0')
-        if not self.options.with_system_deps: self.build_requires('gtest/1.11.0', force_host_context=True)
+        if self.options.with_system_deps: return
+        self.build_requires('protobuf/3.21.4')
+        self.build_requires('grpc/1.48.0')
+        self.build_requires('gtest/1.11.0', force_host_context=True)
 
     def requirements(self):
-        if not self.options.with_system_deps: self.requires("abseil/20220623.0")
-        if not self.options.with_system_deps: self.requires("capstone/4.0.2")
-        if not self.options.with_system_deps: self.requires("grpc/1.48.0")
-        if not self.options.with_system_deps: self.requires("outcome/2.2.3")
+        if self.options.with_system_deps: return
+
+        self.requires("abseil/20220623.0")
+        self.requires("capstone/4.0.2")
+        self.requires("grpc/1.48.0")
+        self.requires("outcome/2.2.3")
         if self.settings.os != "Windows":
-            if not self.options.with_system_deps: self.requires("volk/1.3.224.1")
-            if not self.options.with_system_deps: self.requires("vulkan-headers/1.3.224.1")
-            if not self.options.with_system_deps: self.requires("vulkan-validationlayers/1.3.224.1")
+            self.requires("volk/1.3.224.1")
+            self.requires("vulkan-headers/1.3.224.1")
+            self.requires("vulkan-validationlayers/1.3.224.1")
         self.requires("zlib/1.2.12", override=True)
+        self.requires("openssl/1.1.1s", override=True)
```

Comment:
Why was that not needed before?

Reply:
The is currently an ambiguity in the build graph because different dependencies need different version of openssl. This line disambiguates that and forces openssl/1.1.1s an all dependencies.